app/views/subscriptions_management/index.html.erb

```diff
@@ -37,10 +37,12 @@
         <p class='govuk-body'>
           <%= subscription['created_at'].to_datetime.strftime("Created on %-d %B %Y at %-I:%M%P") %>
         </p>
-        <% if subscription['subscriber_list']['description'].present? %>
-          <%= render "govuk_publishing_components/components/govspeak", {} do %>
-            <%= raw(Kramdown::Document.new(subscription['subscriber_list']['description']).to_html) %>
-          <% end %>
+        <% if subscription['subscriber_list']['url'] =~ %r{transition-check/results} %>
```

Comment:
Oh that's nice! TIL

Reply:
I'll assume you're talking about the groovy `%r` syntax, and not the overall hack 😄.

Reply:
You assume correctly!